### PR TITLE
Intersection of Two Linked List

### PR DESCRIPTION
**Pull request by zenithexpo**
Issue #136  
I have written two approach for this problem with proper explanation in comments, one brute force and other optimized approach in Python.
Tested the code, and provided sample testcase as input.

### CODE REVIEW

**Comment by zenithexpo** [comment]
Checks failed after updating comments, should i remove the diagrammatic example from comments.

File: Code/Python/linked_list_intersection.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+            ptr1 = ptr1.next
+            ptr2 = ptr2.next
+        return None
+ 
+
+if __name__ == '__main__':
+	listA = LinkedList()
+	listA.head = Node(3)
+	node2a = Node(7)
+	node3a = Node(2)
```
Comment (plazzy99) [inline Code/Python/linked_list_intersection.py:102]: make your program to take dynamic inputs

**Review by plazzy99 — CHANGES_REQUESTED** [review]
Update the readme for the same

File: Code/Python/linked_list_intersection.py (hunk 1 above)
Comment (zenithexpo) [reply Code/Python/linked_list_intersection.py:141]: Hello, updated the code for dynamic inputs.

**Comment by tarun26091999** [comment]
you require to take input from the user and also link the code to folder.

File: Code/Python/linked_list_intersection.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+#			headB = Node(data)
+#			listB.head = headB
+#			prev = headB
+#		else:
+#			prev.next = Node(data)
+#			prev = prev.next
+#	prev.next = intersect_node
+
+	listA = LinkedList()
+	listA.head = Node(3)
```
Comment (yasharth291) [inline Code/Python/linked_list_intersection.py:139]: take dynamic input not hard coded stuff

File: Code/Python/linked_list_intersection.py [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+        while ptr1 and ptr2:
+            if ptr1==ptr2:
+                return ptr1
+            ptr1 = ptr1.next
+            ptr2 = ptr2.next
+        return None
+ 
+
+if __name__ == '__main__':
+# uncomment below lines of code to manually enter input of user's choice.
```
Comment (yasharth291) [inline Code/Python/linked_list_intersection.py:99]: remove this comments

**Review by yasharth291 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**